tral/tral_configuration/config.ini

```diff
@@ -8,14 +8,14 @@ sequence_type = AA
         DNA = Phobos, TRED, T-REKS, TRF, XSTREAM
     [[repeat_detector_path]]
         # If the executable is in the system path, supply its name. Otherwise, supply the full path to the executable. Details are explained in TRAL's online docs.
-        PHOBOS = phobos
+        PHOBOS = phobos_*
         HHrepID = hhrepid_64
         HHrepID_dummyhmm = /path/to/home/.tral/data/hhrepid/dummyHMM.hmm
         T-REKS = T-REKS
         TRED = tred
-        TRF = trf
+        TRF = trf409.linux64
```

Comment:
Instead, let's add a symlink in the TRF install script like `ln -s trf409.linux64 /usr/local/bin/trf` so the name is standard. This could be done for phobos too.

Reply:
Thank you, I've integrated it.